### DISCUSSION TITLE
Is this toolkit support pg11?

### DISCUSSION THREAD

**Discussion by lukegao** [timescale/timescaledb-toolkit#284] [Q&A] [answered]
Hi team, is this project support Postgres 11? As I tried and failed to build an docker image with pg11.

**Comment by WireBaron** [comment, marked as answer]
@lukegao We require PG version 12 or higher.  Unfortunately there was a breaking change to function API between PG11 and 12 which would make supporting both versions very painful.

**Reply by lukegao** [reply]
Thanks.
